### ISSUE TITLE
amenity=nursing_home rendered like school/kindergarten

### ISSUE THREAD

**Issue by Prince-Kassad** [gravitystorm/openstreetmap-carto#3848]
Buildings with amenity=nursing_home have their name rendered identically to amenity=school/kindergarten (italics, school-yellow color), with the addition of an amenity-dot for whatever reason.

The equivalent tag combination amenity=social_facility social_facility=nursing_home, on the other hand, is rendered in amenity-brown, with a proper icon, and with the name not in italics.

The two tags are supposed to have an identical meaning so it's really confusing that their rendering differs so much. A nursing home is more an amenity than a school/kindergarten anyway. They should render the same, if we can't get around to drop amenity=nursing_home (probably a tagging issue).

**Comment by jeisenbe** [comment]
The fill color `@societal_amenities:` `#ffffe5` is used for all kinds of education and healthcare areas, eg, schools, kindergartens, universities, clinics, and hospitals, in addition to `nursing_home`. We could certainly start rendering `social_facility=nursing_home` in that style, though we might want to reconsider how to render the other types of `amenity=social_facility` as well.

**Comment by jeisenbe** [comment]
This should be a relatively easy issue for a new contributor to fix, but we will need to discuss what color the icon should be: use the `societal_amenities` color or keep `@public_service` (brown)?

Some test renderings would be helpful.

**Comment by polarbearing** [comment]
IIRC rendering the name for amenity=nursing_home was a compromise as there is lots of legacy tagging, not yet follwowing the structured amenity=social_facility, social_facility=nursing_home approach.

**Comment by jragusa** [comment]
`amenity=nursing_home` is considered as a deprecated tags on the [wiki](https://wiki.openstreetmap.org/wiki/Tag:social_facility%3Dnursing_home)

**Comment by jeisenbe** [comment]
I don't see mention of deprecating `amenity=nursing_home` in the proposal for social_facility: https://wiki.openstreetmap.org/wiki/Proposed_features/social_facility - sometimes these notices get added to the wiki without discussion. And `amenity=nursing_home` is still nearly twice as common as `social_facility=nursing_home`.

**Comment by polarbearing** [comment]
Story described better there now. 
Please note that with the acceptance of the "social_facility" key in 2010 the initial idea was to subtag nursing homes as group_homes with third-level subtagging of the target group. This was later improved with the social_facility=nursing_home value. However the majority of nursing homes still camouflages as group_home, which is not wrong but less specific. Thus the correct comparison is 13.9k old-style amenity=nursing home, vs. 40k = 8.3k social_facility=nursing_home + 32k social_facility=group_home.

**Comment by jeisenbe** [comment]
I suggest removing the rendering of `social_facility=nursing_home` in #4123 because we already support the more common tag `amenity=nursing_home`

**Comment by polarbearing** [comment]
@jeisenbe, this is incorrect. As explained above, the most frequent tagging of nursing homes is still "amenity=social_facility + social_facility=group_home + social_facility:for=senior". due to the flaw in the 2010 social_facility proposal. 
That was fixed with the introduction of social_facility=nursing_home in 2017, which is steadily and organically growing since, 11K now. It would be ridiculous to remove the more modern tagging from rendering, and send the wrong signals.

**Comment by jeisenbe** [comment]
In the USA, a "nursing home" or more precisely a "skilled nursing facility" is not the same thing as a retirement home or group home or "residential home" for seniors.

Compare: https://www.dewis.wales/types-of-residential-care-home

"Nursing homes:

A nursing home is a residential home which offers NHS-funded nursing care in addition to personal care. If your needs are primarily health needs, then you probably need to go into a nursing home rather than an ordinary residential care home.

A nursing home must have a qualified nurse on duty at all times. There is usually a higher ratio of staff to residents to reflect the higher needs of residents."

"Residential care homes:

Residential care homes provide a safe environment with care staff on hand 24 hours a day. Residents receive help with washing, dressing, going to the toilet, eating and drinking, depending upon their care and support needs. Meals are provided.

Your health needs are met by visiting professionals, for example, your GP and a community nurse. There is no requirement for residential care homes to have a qualified nurse on the premises at all times."

**Comment by jeisenbe** [comment]
Do you know where that big import happened? In Wales, these are some of the examples of places mapped as "amenity=social_facility + social_facility=group_home + social_facility:for=senior" - many are residential care homes, not nursing homes.

Here are the first few I found in Wales:

http://www.openstreetmap.org/way/352400991
name=Heatherslade Residential Home
A residential care home: http://theheatherslade.co.uk

http://www.openstreetmap.org/way/352078382
The Newton Grange Residential Home
This is a Residential care home: https://www.carehome.co.uk/carehome.cfm/searchazref/10005025NEWG

http://www.openstreetmap.org/way/518086557
Awel Y Mor Nursing Home
This one appears to be a nursing home
https://www.barchester.com/home/awel-y-mor-care-centre

http://www.openstreetmap.org/way/504111999
Llys-y-Werin
This looks like mainly assisted living (more independent):
http://www.housingcare.org/housing-care/facility-info-14085-llysywerin-gorseinon-wales.aspx

Here's some in SE England: (again searching for  "amenity=social_facility + social_facility=group_home + social_facility:for=senior")

http://www.openstreetmap.org/node/320610908
http://www.redcotlodge.com 
"Residential care home"
Redcot Lodge

http://www.openstreetmap.org/way/288440764
Highfield Residential Home
https://highfieldbirchington.com
Looks like residential care but not 24 hour nursing care.

http://www.openstreetmap.org/way/686097845
Whitebirch Lodge
https://www.carehome.co.uk/carehome.cfm/searchazref/10001050WHIA
"Care Home only (Residential Care)"

**Comment by jeisenbe** [comment]
Admittedly, in Wales and SE England, the usage of `amenity=nursing_home` overlaps: many of these features are also residential care, not 24 hours nursing care, but there are a larger percentage of official "nursing homes" under that tag. And here in Oregon, it looks like more than half of the nursing_home features are skilled nursing facilities, rather than assisted living.
